solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -229,7 +229,7 @@ function openldap_requiremenrs(){
 
 function execute_prereq(){
     echo "Pulling latest CORTX-ALL image"
-    docker pull $CORTX_IMAGE || echo "Failed to pull $CORTX_IMAGE"
+    docker pull $CORTX_IMAGE || echo "Failed to pull $CORTX_IMAGE" && exit 1
```

Comment:
&& takes return code for first command. So it will be called if first command is success. 
e.g. 
```
[root@ssc-vm-rhev4-0707 ~]# docker pull $CORTX_IMAGE || echo "Failed to pull $CORTX_IMAGE" && echo "exit 1"
2.0.0-5044-custom-ci: Pulling from seagate/cortx-all
Digest: sha256:41efed26f363528c4a77bb0f3ba6352d8d4dfe1669ec9f914a9d3045f76549da
Status: Image is up to date for cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5044-custom-ci
cortx-docker.colo.seagate.com/seagate/cortx-all:2.0.0-5044-custom-ci
exit 1
[root@ssc-vm-rhev4-0707 ~]#
```

Reply:
Done